core/runtime/dynamic_map_internal.odin

```diff
@@ -300,17 +302,36 @@ __dynamic_map_set :: proc(h: Map_Header, hash: Map_Hash, value: rawptr, loc := #
 
 	if __dynamic_map_full(h) {
 		__dynamic_map_grow(h, loc)
-		// index = __dynamic_map_find(h, hash).entry_index
-		// assert(index >= 0)
 	}
 	
 	return __dynamic_map_get_entry(h, index)
 }
 
 
+@(private="file")
+next_pow2 :: proc "contextless" (n: int) -> int {
+	n := n
+	if n <= 0 {
+		return 0
+	} else if n <= 2 {
+		return n
+	}
+	n -= 1
```

Comment:
If replaced thusly, it does produce 16 for 8, 32 for 16, and so on:
```cpp
if intrinsics.count_ones(n) != 1 {
    n -= 1
}
```

Reply:
Oh, I see. The `next_pow2` is only for the initial reservation, and it just doubles after that. I withdraw my objections.